openapi-generator-rh-cpp-unreal/src/main/java/com/rallyhere/codegen/RhCppUe4Generator.java

```diff
@@ -183,6 +184,9 @@ public RhCppUe4Generator() {
         importMapping.put("TSet", "#include \"Containers/Set.h\"");
         importMapping.put("TVariant", "#include \"Misc/TVariant.h\"");
 
+        apiNameMapping = new HashMap<>();
+        apiNameMapping.put("File", "RemoteFile");
```

Comment:
If we end up doing this more often, we probably want to make this a cmdline input to the generator.  But for this, this is fine

Reply:
I briefly looked around for how the other commandline args we have do this and they are all single values not maps.  The generator has some map commandline options but they are pretty verbose (something like `--apinamemapping File=RemoteFile` for each value in the map)